lib/modules/platform/gitea/utils.ts

```diff
@@ -137,3 +137,33 @@ export function toRenovatePR(data: PR, author: string | null): Pr | null {
     hasAssignees: !!(data.assignee?.login ?? is.nonEmptyArray(data.assignees)),
   };
 }
+
+/**
+ * Check if a repository is usable.
+ * A repo isn't usable:
+ * - if it's a mirror
+ * - if we don't have push permissions
+ * - if it doesn't have pull requests
+ * @param repo Repo to check
+ * @returns  true` if the repository is usable, `false` otherwise
+ */
+export function usableRepo(repo: Repo): boolean {
+  if (repo.mirror === true) {
+    return false;
+  }
+
+  if (repo.permissions.pull === false || repo.permissions.push === false) {
+    logger.debug(
+      `Skipping repository ${repo.full_name} due to lack of pull or push permissions`,
+    );
+    return false;
+  }
+
+  if (repo.has_pull_requests === false) {
+    logger.debug(
+      `Skipping repository ${repo.full_name} due to lack of pull requests`,
```

Comment:
I suspect you meant:

> Skipping repository `name` because it does not support pull requests

Or something similar to that? It seems to me the important bit is that Renovate needs the repo/platform to support/allow pull requests to opens its PRs.

Reply:
the platform supports it, but you can disable PR's in repo settings